### ISSUE TITLE
mobile: update to cli@3.16.7 and cut new releases

### ISSUE THREAD

**Issue by bassosimone** [ooni/probe#2376]
This issue is about updating mobile to 3.16.7 (see https://github.com/ooni/probe/issues/2375) and cutting new releases.

**Comment by bassosimone** [comment]
I updated to cli@3.16.7 in https://github.com/ooni/probe-ios/pull/503 and https://github.com/ooni/probe-android/pull/547. Now we need to cut releases.

**Comment by bassosimone** [comment]
So, we're checking why there are missing tags. That said, by manually checking the repository history, I could confirm that we included a fix to use probe-cli@3.16.7 in https://github.com/ooni/probe-ios/commit/1a36ed349c2fbad6442f8db1436c6942a4db6c8a and that we bumped the probe-ios version to 3.7.3 in https://github.com/ooni/probe-ios/commit/509b1009dd829146e46091e47905e6bcf0eebfca. So, we're probably fine as far as this issue is concerned. The missing tags issue will be dealt with separately.
